src/notes/note.entity.ts

```diff
@@ -89,6 +90,14 @@ export class Note {
   @JoinTable()
   tags: Promise<Tag[]>;
 
+  @Column({
+    default: 2,
```

Comment:
I think this redundant

Reply:
It is, but this statement cannot be removed, otherwise the app crashes on startup with a constraint violation.